### ISSUE TITLE
Standalone: Let the user configure dip switches

### ISSUE THREAD

**Issue by francisdb** [vpinball/vpinball#1210]
**Is your feature request related to a problem? Please describe.**

[Victory](https://vpuniverse.com/files/file/10390-victory-gottlieb-1987/) has below code which is called on startup and causes standalone vbscript to report a runtime exception in `core.vbs` `AddForm` `mLWF.AddCtrl`

Seems to be the only table doing this on startup to pre-populate the `Options` value

```vbscript
Dim vpmDips1: Set vpmDips1 = New cvpmDips
	With vpmDips1
	 	.AddForm 350, 380, "Victory - Table Options"
	 	.AddLabel 0,0 + DT_offs,180,15,"Play Options"
	 	.AddFrameExtra 15,15 + DT_offs,205,"Left Outlane*",&H00003000, Array("Liberal", 0, "Normal", &H00001000, "Difficult", &H00002000)
	 	.AddChkExtra 0,150 + DT_offs,155, Array("Disable Menu Next Start", &H00000001)
	End With
```

**Describe the solution you'd like**
Let the user configure dips on standalone

**Comment by jsm174** [comment]
Unfortunately adding support for this would be extremely difficult considering this generates a Windows UI.

I guess this would all have to be recreated in ImGui..

**Comment by francisdb** [comment]
Another affected table is https://vpuniverse.com/files/file/8359-playboy-stern-2002/

**Comment by francisdb** [comment]
@vbousquet these menus already look quite ugly on windows when you enable display scaling. It would be useful for windows as well. Implementing `mLWF` to show something like the tweak menu would probably work?

**Comment by francisdb** [comment]
For now you can use [vpxtool](https://github.com/francisdb/vpxtool) for this, uses data from [pinmame-nvram-maps](https://github.com/tomlogic/pinmame-nvram-maps)

**Comment by vbousquet** [comment]
I agree that the existing vbs implementation is obsolete. My aim on this is to implement the pinmame nvram maps in the plugin and have the PinMAME plugin to expose them as options. But there are too many more important things to be done before.
